src/library/trackmodel.h

```diff
@@ -147,6 +147,10 @@ class TrackModel {
     virtual TrackModel::CapabilitiesFlags getCapabilities() const {
         return TRACKMODELCAPS_NONE;
     }
+    virtual bool hasCapabilities(const TrackModel::CapabilitiesFlags& flags) const {
```

Comment:
Please prefer to pass those trivially copyable types by value instead of by const-ref.